.eslintrc

```diff
@@ -38,6 +38,7 @@
     "react/no-unused-prop-types": "off",
     "react/prefer-stateless-function": "off",
     "react/prop-types": ["error", { "skipUndeclared": true }],
+    "react/forbid-prop-types": "off",
```

Comment:
What about `"react/forbid-prop-types": ["on", { "forbid": "any" }]`?

Reply:
Yes. Sure. Missed that.